.github/DEV_GUIDE.md

```diff
@@ -86,6 +86,8 @@ Now, let's connect the main goonstation repository to your client.
 
 We need to add the main Goonstation repository as a remote now. :satellite:
 
+It is likely that when you cloned your github repository, this upstream remote was already set up for you. If that's the case, you can skip adding the remote, and move straight to setting your local master to track the main branch.
```

Comment:
```suggestion
It is likely that when you cloned your github repository, this upstream remote was already set up for you. If that's the case, you can skip adding the remote, and move straight to setting your local master to track the master branch.
```
technically master is the 'main' branch but considering a ton of repos have a branch called `main` this seems clearer